src/pregenerated.rs

```diff
@@ -46,7 +81,7 @@ impl ftdi_parity_type {
 }
 #[repr(transparent)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
-pub struct ftdi_parity_type(pub u32);
```

Comment:
These changes look a bit strange, not sure where they are coming from.

Reply:
Looks like this is system-dependent, see rust-lang/rust-bindgen#1966.

Reply:
This is caused by a lack of a bindgen::Builder::default_macro_constant_type setting. I can submit another PR to fix.

Reply:
Already fixed, sorry :-)